onboard/catkin_ws/src/system_utils/scripts/topic_transforms.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+import math
+
+from transforms3d.euler import quat2euler
+
+import rospy
+from geometry_msgs.msg import Pose, Twist, Vector3
+from nav_msgs.msg import Odometry
+from sensor_msgs.msg import Imu
+
+
+# Class to store data for topic transformations
+class TopicTransformData:
+    def __init__(self, input_topic, input_type, input_type_conversion, output_topic, output_type,
+                 output_type_conversion, subscriber=None, publisher=None, publisher_queue_size=1):
+        self.input_topic = input_topic
+        self.input_type = input_type
+        self.input_type_conversion = input_type_conversion
+        self.output_topic = output_topic
+        self.output_type = output_type
+        self.output_type_conversion = output_type_conversion
+        self.subscriber = subscriber
+        self.publisher = publisher
+        self.publisher_queue_size = publisher_queue_size
+
+
+# Class to store conversion functions
+class Conversions:
+
+    @staticmethod
+    def quat_to_vector(quat_msg):
+        # Convert quaternion to euler angles
+        euler_angles = quat2euler(
+            [quat_msg.w, quat_msg.x, quat_msg.y, quat_msg.z], axes='sxyz')
```

Comment:
```suggestion
        euler_angles = quat2euler([quat_msg.w, quat_msg.x, quat_msg.y, quat_msg.z])
```

Since 'sxyz' is the default for `axes`, and I don't think there would be a use case for any other axis sequence.